docs/src/dMCDA.md

```diff
@@ -1 +1,159 @@
+
+$$R_i = \textbf{max}_{j} \frac{PIS_j-X_{i,j}}{PIS_j-NIS_j}$$
+
+I.e. this gives a measure of the worst hidden trade-off in the Manhattan Distance (which is not considered in TOPSIS). The final ranking $Q$ is calculated from a linear combination of $R$ and $S$ where the weighting of $R$ and $S$ are determined by the balance of ‘group utility’ (measured by $S$) and ‘individual regret’ (measured by $R$) a decision maker wants to incorporate in their decision solution.
+
+$$S^- = \textbf{min}_{i}(S_i),S^+ = \textbf{max}_{i}(S_i)$$
+$$R^- = \textbf{min}_{i}(R_i),R^+ = \textbf{max}_{i}(R_i)$$
+$$Q_i = \delta \frac{S_i-S^-}{S^+-S^-} + (1-\delta)\frac{R_i-R^-}{R^+-R^-}$$
+
+Here, $\delta$ is a weighting which balances strategy between maximising all criteria and regret due to the worst performing criteria. Additional steps to the algorithm give guidelines for choosing between a single best alternative and n (n>1) best alternatives.
+
+## Post-ranking distance sorting 
+If highly ranked sites happen to be very close together in a particular domain, this can represent an issue for site selection. This is because the site selection algorithm will seed or shade in a very small region, representing a significant loss of resources if a high damage, localised event, such as a ship grounding, were to occur. To represent an insurance policy against this possibility, the user can also specify distance thresholds for selected sites. If sites are selected which are too close, sites will be replaced by the next highest ranked sites which are further apart until the distance threshold is satisfied. The distance sorting parameter inputs are described in the table below:
+
+|Parameter                    |Description                                                                        |
+|-----------------------------|-----------------------------------------------------------------------------------|
+|```dist_thresh```            |Sites closer than (median distance between sites) - dist_thresh% will be replaced. |
+|```top_n```                  |Number of top ranked alternatives to select from when replacing sites.             |
+
+
+## Using site selection separately
+ADRIA's site selection algorithms can also be implemented outside of the ecological model, using the `run_site_selection()` function. The user provides data layers representing the selection criteria and weights to produce a set of site ranks for seeding and/or shading. An example of the function's usage is shown below. First an ADRIA domain is loaded, containing environmental data such as wave and heat stress layers and site data. Then a criteria data frame is created, which specifies criteria weightings and wave and dhw scenarios. Finally, coral cover layers for each scenario are loaded. These are used within `run_site_selection()` to generate reccommended intervention sites for seeding at time `ts`, for each parameter scenario.
```

Comment:
> ADRIA's site selection algorithms can also be implemented outside of the ecological model

"implemented" ->"applied"

To me, "implemented" implies users are re-creating the algorithms.

"outside of the" -> "separately from"

Reply:
> Then a criteria data frame is created

data frame -> DataFrame

Reply:
> First an ADRIA domain is loaded, containing environmental data such as wave and heat stress layers and site data.

Add a dummy link at the end of the sentence as a reminder to cross-reference the relevant page with more detail in the documentation.

Reply:
> Then a criteria data frame is created, which specifies criteria weightings and wave and dhw scenarios.

It doesn't hold just criteria though.

If you need one, why not write a function that returns just the set of criteria values?

Later, you need to qualify use of the DataFrame (with `criteria_df[criteria_df.guided.>0, :]`, which is unexplained in the snippet btw) so with a criteria-specific you can avoid having to put that bandaid on.

Reply:
I'll create an issue for this change as it's not a documentation change

Reply:
Hey @Rosejoycrocker, to clarify, would this function be like a sampler just for site selection criteria? Trying to work out inputs etc

Reply:
Yes, samples drawn from just the selection criteria factors.

> Trying to work out inputs etc

Please leverage Julia's multiple dispatch. You just need to define a `sample` function that takes in the Criteria type and the number of desired samples. See the existing `sample` function and the docs for [ModelParameters.jl](https://github.com/rafaqz/ModelParameters.jl).

Reply:
Thanks for the clarification. I've addressed some changes you've requested in comments, but I'll work on the sampling function first before coming back to the rest.